File: Algorithms/119. Pascal's Triangle II/javascript/solution.dp.js
/**
 * @param {number} rowIndex
 * @return {number[]}
 */
var getRow = function (rowIndex) {
    let curr = [],
        prev = [1];
    for (let i = 1; i <= rowIndex; i++) {
        curr = [1];
        for (let j = 1; j < i; j++) {
            curr[j] = prev[j - 1] + prev[j];
        }
        curr.push(1);
        prev = curr;
    }
    return prev;
};

File: Algorithms/119. Pascal's Triangle II/golang/solution.dp.go
func getRow(rowIndex int) []int {
	var curr []int
	var prev = []int{1}
	for i := 1; i <= rowIndex; i++ {
		curr = []int{1}
		for j := 1; j < i; j++ {
			curr = append(curr, prev[j-1]+prev[j])
		}
		curr = append(curr, 1)
		prev = curr
	}
	return prev
}
